### ISSUE TITLE
Purple Vectors

### ISSUE THREAD

**Issue by Automa-C3PO** [JACoders/OpenJK#1085]
Info: Windows 7 64bit Professional, singleplayer.
A weird bug occuring with newer and older versions of OpenJK. A crossing net of linked purple lines appear among enemy NPC's, whenever the cheat command **helpusobi** is activated. It disappears when they're idle and not alerted. It seems an AI debugging issue left by developers. These random pictures show the lines:
![jasp1](https://user-images.githubusercontent.com/82467855/114603376-f0454b80-9c97-11eb-90b1-e42d10acc2b6.jpg)
![jasp2](https://user-images.githubusercontent.com/82467855/114603408-f89d8680-9c97-11eb-9825-9e9b74b9d99d.jpg)

**Comment by ensiform** [comment]
It seems you might have an additional cvar activated but I'm not sure which one off hand. As helpusobi 1 by itself does not turn this on.

You could try making a backup of your openjk_sp.cfg and then run cvar_restart to reset most to defaults. Then bring yours back to figure out which one is the culprit.

**Comment by Automa-C3PO** [comment]
Darn...I got it! Thanks _ensiform_, I reset the parameters (c++variables?) as you said, I tried in blocks what could be the culprit, after many attempts it turned up being a debug option hidden in this command: **d_npcAI**. When **set to 4 or higher**, it displayed them. I probably changed it after a modded autoexec and mistakenly overwrote with the same value. No apparent alteration was brought to AI, except these blasted vectors. Not sure whether and how this affects the behaviours...the lines linked all enemy NPC's in the same squad. This means that if they moved or attacked, it showed the hierarchy between them and dinamically changed if player killed some, so if one neutralizes an officer say, the game reassigns a new one and they keep tactics. The highest in rank often had the most on. None of Sith had ever any. They seem to use another kind of AI indipendently, sometimes acting as team but not like other "minions". _I declare it solved_, don't put `d_npcAI "4"` or higher in configuration files.
